ci/README.md

```diff
@@ -8,6 +8,13 @@ image at [`lyft/envoy:<hash>`](https://hub.docker.com/r/lyft/envoy/) is an image
 corresponds to the master commit at which the binary was compiled. Lastly, `lyft/envoy:latest` contains
 an Envoy binary built from the latest tip of master that passed tests.
 
+# Using alpine envoy image
```

Comment:
@junr03 Thanks, I've changed the section organization to have two subsections for Ubuntu and Alpine images.